### PR TITLE
Added design resources page

### LINKED ISSUE #35: Create a page for design collaboration and shared design resources
I think a page like this could help us with two things.

First, it looks like we will be using Figma for many designs due to the ease of sharing and collaboration. While Github has many pre-defined processes built-in, Figma and other design tools do not. So I think it would be helpful if we articulate these for us.

Second, there are a few design resources being worked on that we will make publicly available for anyone to use and contribute to. Currently we have many Figma links floating around with no way to find them again other than asking around or searching Slack/Figma history. We can also list important design resources on this page.

This page would likely be under "Resources".

Below is a mock-up with some quickly written copy. What do you think of this? Note that this is heavily slanted towards collaborating on Figma community files. So this might need to be a sub-page under a more general "Design collaboration" or "Open design" section that is tool-agnostic.

Design collaboration page 201019

### PR DESCRIPTION
**Pull request by GBKS**
Based on #35, with slight updates to copy and resources, as the original issue is pretty old by now.

[Preview](https://deploy-preview-413--sad-borg-390916.netlify.app/guide/designing-products/design-resources/)

What's weird is that all the resource files on the page are created by me. That's not cool and we should add work by others. What can we add?

Closes #35.

### CODE REVIEW

**Comment by pavlenex** [comment]
> What can we add?

Art meets Botoin by @AlexaAker 

https://www.figma.com/file/MQ7XU3Xy2e03aBOxk50RtY/Art-Meets-Bitcoin---3D?node-id=10%3A17

**Comment by GBKS** [comment]
Added @AlexaAker's file. Great idea.

File: guide/designing-products/design-resources.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+
+
+-->
+
+# Collaboration on design source files
+
+**The community maintains several design files for this guide and related projects. You can duplicate, use and modify them as you like.**
+
+There is no design tool yet that provides a public, open, and communal collaboration platform. At the moment many of us use Figma, as it comes closest. You can also use other design tools and workflows of your choice.
```
Comment (pavlenex) [inline guide/designing-products/design-resources.md:61]:
```suggestion
There is no design tool yet that provides public, open, and communal collaboration platform. At the moment, many of us use Figma, as it comes closest. You can also use other design tools and workflows of your choice.
```

File: guide/designing-products/design-resources.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+During this process, it is always good to be in communication with the file owner and other contributors.
+
+## Collaborating
+
+Figma has several useful collaboration features. They do not force a specific workflow, so it may not be always clear what the best way is to work together. Here are some tips:
+
+### Sharing
+
+Sharing is one of the most powerful features of Figma, as it works on all platforms as well as in the browser, and does not require accounts for only viewing design files. Real-time collaboration allows multiple users to work on the same file simultaneously.
```
Comment (pavlenex) [inline guide/designing-products/design-resources.md:99]:
```suggestion
Sharing is one of the most powerful features of Figma, as it works on all platforms as well as in the browser and does not require accounts for only viewing design files. Real-time collaboration allows multiple users to work on the same file simultaneously.
```

**Comment by pavlenex** [comment]
Curious, is this the `assets` page we've been talking about?

**Comment by GBKS** [comment]
@pavlenex pretty much, yes. It's a list of Figma files people can freely use to kick off their work, along with some good practice on how to collaborate and contribute to design files.

**Review by pavlenex — APPROVED** [review]

**Pull request merged**